data/json/mapgen/house_garage_prepper.json

```diff
@@ -101,26 +101,26 @@
                 { "point" : "trap",    "id" : "tr_shotgun_2",       "x" : [ 8, 8], "y" : [14,14] },
                 { "point" : "trap",    "id" : "tr_shotgun_2",       "x" : [ 7, 7], "y" : [ 2, 2] }
             ],
-            "place_items": [
-                { "item" : "bed",            "chance" :  80, "x" : [13,14], "y" : [20,21] },
-                { "item" : "bedroom",        "chance" :  75, "x" : [ 6,21], "y" : [13,20], "repeat" : [1,2] },
-                { "item" : "livingroom",     "chance" :  80, "x" : [ 1,12], "y" : [ 2, 6], "repeat" : [1,2] },
-                { "item" : "homeguns",       "chance" :  90, "x" : [12,12], "y" : [21,21] },
-                { "item" : "dresser",        "chance" :  50, "x" : [21,21], "y" : [21,21] },
-                { "item" : "kitchen",        "chance" :  80, "x" : [ 6, 7], "y" : [ 5, 6], "repeat" : [1,5] },
-                { "item" : "cannedfood",     "chance" :  90, "x" : [11,14], "y" : [13,13], "repeat" : [1,4] },
-                { "item" : "tools_common",   "chance" :  40, "x" : [14,16], "y" : [11,11] },
-                { "item" : "supplies_fuel",  "chance" : 100, "x" : [14,14], "y" : [ 3, 6], "repeat" : [1,2] },
-                { "item" : "mischw",         "chance" :  90, "x" : [14,16], "y" : [11,11] },
-                { "item" : "tools_gunsmith", "chance" : 100, "x" : [17,21], "y" : [11,11], "repeat" : [1,3] },
-                { "item" : "ammo_parts",     "chance" : 100, "x" : [21,21], "y" : [ 6,11], "repeat" : [2,6] },
-                { "item" : "ammo_casings",   "chance" : 100, "x" : [21,21], "y" : [ 6,11], "repeat" : [1,3] },
-                { "item" : "gunmod_common",  "chance" :  40, "x" : [21,21], "y" : [ 6,11] },
-                { "item" : "guns_survival",  "chance" :  70, "x" : [15,16], "y" : [ 8, 9], "magazine" : 100, "ammo" : 75 },
-                { "item" : "fridge",         "chance" :  50, "x" : [ 4, 4], "y" : [11,11] },
-                { "item" : "oven",           "chance" :  50, "x" : [ 3, 3], "y" : [11,11] },
-                { "item" : "softdrugs",      "chance" :  75, "x" : [ 2, 4], "y" : [13,16] },
-                { "item" : "cleaning",       "chance" :  75, "x" : [ 2, 4], "y" : [13,16] }
+            "place_loot": [
+                { "group" : "bed",            "chance" : 95, "x" : [13,14], "y" : [20,21], "repeat" : [1,3] },
+                { "group" : "bedroom",        "chance" : 85, "x" : [ 6,21], "y" : [13,20], "repeat" : [1,4] },
+                { "group" : "livingroom",     "chance" : 90, "x" : [ 1,12], "y" : [ 2, 6], "repeat" : [1,5] },
+                { "group" : "guns_common",    "chance" : 75, "x" : [12,12], "y" : [21,21], "magazine" : 100, "ammo" : 100 },
```

Comment:
It's not essential by any means. I can't see any other problems so I'll try merge-testing it.